v4/export/config.go

```diff
@@ -117,6 +127,28 @@ func (conf *Config) GetDSN(db string) string {
 	return dsn
 }
 
+func (config *Config) createExternalStorage(ctx context.Context) (storage.ExternalStorage, error) {
+	path := config.OutputDirPath
+	var b *backup.StorageBackend
+	var err error
+	options := &config.BackendOptions
+	failpoint.Inject("ExternalStorageEndpoint", func(val failpoint.Value) {
+		// used for testing
+		endpoint := val.(string)
+		if endpoint != "" {
+			options.S3.Endpoint = endpoint
+			options.S3.ForcePathStyle = true
+			options.GCS.Endpoint = endpoint
+		}
+	})
+	b, err = storage.ParseBackend(path, options)
+	if err != nil {
+		return nil, err
+	}
+	// TODO support sendCreds config option?
```

Comment:
no need to support Send Creds since Dumpling works independently.

Reply:
so we should pass false?

Reply:
any value is fine since it won't be used. i mean there's nothing "TODO" here 😄